duracloud/duracloud
# DC-1351: Support for extended manifest attributes

**JIRA Ticket**: https://duracloud.atlassian.net/browse/DURACLOUD-1351

# What does this Pull Request do?
* Add extended tsv option for manifests with additional attributes
* Remove ContentMessage and related methods as it was unused

# How should this be tested?

* Redeploy duradmin
* Select a space
* Generate a manifest using the extended-tsv option
* See that the manifest contains `content-size` and `mime-type` 

# Additional Notes:

Some alt text might be nice in the drop down options to explain what they contain.

I also didn't really get to test this as I haven't deployed Duracloud in a while.

# Interested parties
@duracloud/committers
